packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/dataservice/ProjectController.java

```diff
@@ -567,31 +570,70 @@ public ResponseEntity<ProjectExport> exportProject(@PathVariable("id") final UUI
 	@ApiResponses(
 			value = {
 				@ApiResponse(
-						responseCode = "200",
+						responseCode = "201",
 						description = "The project export",
 						content = {
 							@Content(
 									mediaType = "application/json",
 									schema =
-											@io.swagger.v3.oas.annotations.media.Schema(
-													implementation = ProjectExport.class))
+											@io.swagger.v3.oas.annotations.media.Schema(implementation = Project.class))
 						}),
 				@ApiResponse(
 						responseCode = "503",
 						description = "An error occurred when trying to communicate with either the postgres or spicedb"
 								+ " databases",
 						content = @Content)
 			})
-	@PostMapping("/import")
+	@PostMapping(value = "/import", consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
 	@Secured(Roles.USER)
-	public ResponseEntity<Project> importProject(@RequestBody final ProjectExport projectExport) {
+	public ResponseEntity<Project> importProject(@RequestPart("file") final MultipartFile input) {
+
+		ProjectExport projectExport;
+		try {
+			projectExport = objectMapper.readValue(input.getInputStream(), ProjectExport.class);
+		} catch (final Exception e) {
+			log.error("Error parsing project export", e);
+			throw new ResponseStatusException(HttpStatus.BAD_REQUEST, messages.get("projects.import-parse-failure"));
+		}
+
+		final String userId = currentUserService.get().getId();
+		final String userName = userService.getById(userId).getName();
+
+		Project project;
 		try {
-			return ResponseEntity.ok(cloneService.importProject(projectExport));
+			project = cloneService.importProject(userId, userName, projectExport);
 		} catch (final Exception e) {
 			log.error("Error importing project", e);
 			throw new ResponseStatusException(
 					HttpStatus.SERVICE_UNAVAILABLE, messages.get("postgres.service-unavailable"));
 		}
+
+		try {
+			project = projectService.createProject(project);
+		} catch (final Exception e) {
+			log.error("Error creating project", e);
+			throw new ResponseStatusException(
+					HttpStatus.SERVICE_UNAVAILABLE, messages.get("postgres.service-unavailable"));
+		}
+
+		try {
+			final RebacProject rebacProject = new RebacProject(project.getId(), reBACService);
+			final RebacGroup rebacAskemAdminGroup = new RebacGroup(ReBACService.ASKEM_ADMIN_GROUP_ID, reBACService);
+			final RebacUser rebacUser = new RebacUser(userId, reBACService);
+
+			rebacUser.createCreatorRelationship(rebacProject);
```

Comment:
Good remembering this - though I imagine this became obvious when you could not see the project after importing it :)